src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -548,3 +554,58 @@ def subset_results_by_state(processed_results_dir: str, state: str) -> None:
             )
             write_to_disk(state_data, output_file_path)
         logger.info(f"Finished writing {observer} files for {state_name}.")
+
+
+def write_shard_metadata(
+    observer: str, obs_data: pd.DataFrame, obs_dir: Path, seed: str
+) -> None:
+    # Writes metadata for each shard of data. This will be proportions available to noise
+    # for specific columns.
+
+    def _get_metadata_values(df: pd.DataFrame, location: str):
+        metadata_df = pd.DataFrame(index=[location])
+        metadata_df["number_of_rows"] = len(df)
+        for column in METADATA_COLUMNS:
```

Comment:
Wouldn't it be better to import `pseudopeople` and use the `pseudopeople` metadata to determine which columns have the copy from within household and nicknames noise functions? That would mean that we'd keep this in sync with the code from pseudopeople.

Reply:
Yes - so move this to pseudopeople metadata then?

Reply:
I _think_ what Rajan is referring to is the pseudopeople.metadata::COPY_HOUSEHOLD_MEMBER_COLS? 

But it doesn't really match what you're doing here either so idk.
https://github.com/ihmeuw/pseudopeople/blob/main/src/pseudopeople/constants/metadata.py#L86-L96